PyExpLabSys/drivers/keithley_2000.py

```diff
@@ -0,0 +1,121 @@
+""" Simple driver for Keithley 2000 DMM """
+from PyExpLabSys.drivers.scpi import SCPI
+
+
+class Keithley2000(SCPI):
+    """
+    Simple driver for Keithley 2000 DMM
+    """
+
+    def __init__(self, interface, hostname='', device='',
+                 baudrate=9600, gpib_address=None):
+        if interface == 'serial':
+            SCPI.__init__(self, interface=interface, device=device,
+                          baudrate=baudrate, line_ending='\n')
+            self.comm_dev.timeout = 2
+            self.comm_dev.rtscts = False
+            self.comm_dev.xonxoff = False
+        if interface == 'gpib':
+            SCPI.__init__(self, interface=interface, gpib_address=gpib_address)
+
+    def set_bandwith(self, measurement='voltage:ac', bandwidth=None):
+        scpi_cmd = 'SENSE:{}:DETector:BANDwidth'.format(measurement)
+        if bandwidth is not None:
+            DMM.scpi_comm(scpi_cmd + ' {}'.format(bandwidth))
+        value_raw = DMM.scpi_comm(scpi_cmd + '?')
+        value = float(value_raw)
+        return value
+
+    def set_range(self, value: float):
+        """
+        Set the measurement range of the device, 0 will indicate auto-range
+        """
+        if value > 1000:
+            value = 1000
+        if value < 0:
+            value = 0
```

Comment:
The one place where Labview got the lead, they have a coerce in range function builtin. You can also do the `min(upper_range, max(lower_range, number))` only, its readability sucks!